docs/sections/papers/ultrafeedback.md

```diff
@@ -46,43 +47,66 @@ from distilabel.steps import (
 )
 from distilabel.steps.tasks import TextGeneration, UltraFeedback
 
+sample_three_llms = sample_n_steps(3)
+
 
 with Pipeline(name="ultrafeedback-pipeline") as pipeline:
     load_hub_dataset = LoadHubDataset(
         name="load_dataset",
         output_mappings={"prompt": "instruction"},
+        batch_size=2,
```

Comment:
Does this really matter if we're already setting `input_batch_size` in the follow up steps?

Reply:
Assuming this is only for the `sample_n_steps` function, shouldn't we define the `batch_size` within that function instead of on the previous one?

Reply:
Yes, it matters. The routing batch function, as the name indicates, routes batches. If we have the default `batch_size` of 50, then we will have much less batches, and therefore much less variety in the LLMs that were chosen.

Reply:
Yes, what I meant is whether isn't it more intuitive to define the output batch size of the routing in the `sample_n_steps` function instead, as that function will receive a batch of inputs of N size, but can then route those in mini batches, say of size N // 5, right? I asked more in terms of how intuitive or counterintuitive can be right now.